tests/v2/validation/provisioning/rke2/README.md

```diff
@@ -339,4 +339,13 @@ Please read up on general k8s to get an idea of correct formatting for:
         }
     }
 }
+```
+## SSHTests
+sshTests runs through the list of sshTests defined in ssh.go. CheckCPU verifies that the cpu usage of a node does not go past 100%
```

Comment:
nit; make this 2nd sentence a bullet instead, so that its clear this list will grow